plugins/multiserver/com_osvr_Multiserver_Sensics_zSight.json

```diff
@@ -0,0 +1,18 @@
+{
+  "deviceVendor": "Sensics",
+  "deviceName": "Sensics zSight HMD",
+  "author": "Virendra Singh",
```

Comment:
Convention is to have this be a Git-style author: `Your Name <youremail@example.com>` if you're comfortable with that. If you'd rather not, you can stick Sensics in there with a sensics support email address.